### ISSUE TITLE
feat(a11y): Handle Notifications as Live-Regions so Screenreaders prioritize them / see them at all

### ISSUE THREAD

**Issue by Rudo-hayate** [Rudo-hayate/open-webui-a11y#2]
### Check Existing Issues

- [x] I have searched for all existing **open AND closed** issues and discussions for similar requests. I have found none that is comparable to my request.

### Verify Feature Scope

- [x] I have read through and understood the scope definition for feature requests in the Issues section. I believe my feature request meets the definition and belongs in the Issues section instead of the Discussions.

### Problem Description

Currently, notifications in OpenWebUI do not get picked up by Screenreaders, meaning that the content is not visible to impaired users at all.

<img width="2546" height="897" alt="Image" src="https://github.com/user-attachments/assets/296bc5e3-ae94-4a00-8c25-b89ea80d0858" />

<img width="2284" height="801" alt="Image" src="https://github.com/user-attachments/assets/4b90ed6c-319e-44cb-a8e8-90653447ee88" />

<img width="1904" height="932" alt="Image" src="https://github.com/user-attachments/assets/5fc10820-5255-4f5c-a824-b5c8b7bf0ce1" />

### Desired Solution you'd like

Apply the correct aria-labels to notifications so the content is prioritized and red to the user when they come up.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Guides/Live_regions

### Additional Context

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-4-1-3-statusmeldungen-programmatisch-verfuegbar
